README.md

```diff
@@ -99,11 +99,11 @@ See the [getting started](https://meshery.io/#getting-started) section to quickl
         </tr>
         <tr><td class="beta-adapters"></td></tr>
         <tr>
-          <td rowspan="5" class="alpha-adapters">alpha</td>
+          <td rowspan="4" class="alpha-adapters">alpha</td>
         </tr>
         <tr>
           <td><a href="https://github.com/layer5io/meshery-maesh">
-            <img src='https://github.com/containous/maesh/raw/master/docs/content/assets/img/maesh.png' alt='Maesh Service Mesh' align="middle" hspace="5px" vspace="5px" height="30px" width="30px">Meshery adapter for Maesh</a>
+            <img src='https://github.com/containous/maesh/raw/master/docs/content/assets/img/maesh.png' alt='Maesh Service Mesh' align="middle" hspace="5px" vspace="5px" height="30px" width="30px">Meshery adapter for Mesh</a>
```

Comment:
Thanks for this PR, @UdokaVrede. This particular service mesh is actually spelled "Maesh". - https://containo.us/maesh/

Reply:
Thanks @leecalcote